ethereum/contracts/TestERC20.sol

```diff
@@ -38,4 +38,13 @@ contract TestERC20 is ERC20 {
     function approveInternal(address owner, address spender, uint256 value) public {
         _approve(owner, spender, value);
     }
+
+    // You can send a zero-value transfer directly to the contract address to
+    // get a 100 standard unit tokens.
+    receive() external payable {
+        mint(msg.sender, 100 * 10 ** uint(_decimals));
```

Comment:
I had `10**uint(_decimals)` which I like better, but the solidity formatter separated them.